dlup/utils/mask.py

```diff
@@ -0,0 +1,58 @@
+# coding=utf-8
+# Copyright (c) dlup contributors
+"""Utilities to work with binary masks"""
+import cv2
+import numpy as np
+import shapely.affinity
+from shapely.geometry import Polygon
+
+
+def _DFS(polygons, contours, hierarchy, sibling_id, is_outer, siblings, offset=(0, 0), scaling=1.0) -> list[Polygon]:
+    # Adapted FROM: https://gist.github.com/stefano-malacrino/7d429e5d12854b9e51b187170e812fa4
```

Comment:
Missing docstring

Reply:
Missing input typing

Reply:
Did some. It's not trivial here, later on we will do strict mypy